api/v1/.htaccess

```diff
@@ -7,6 +7,8 @@ php_value include_path ".:../..:/var/www/common:/usr/share/pear"
 
 Header Unset Expires
 SetEnvIf Origin ^(https?://.+\.burningflipside.com(?::\d{1,5})?)$   CORS_ALLOW_ORIGIN=$1
+SetEnvIf Origin ^(https?://localhost(?::\d{1,5})?)$   CORS_ALLOW_ORIGIN=$1
```

Comment:
Probably something like https://www.kerstner.at/2015/02/enabling-cross-origin-resource-sharing-cors-for-php/